deploy-cloud-functions/tests/cleaner/cleanTestFunctions.test.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import { CloudFunctionClient } from '../../src/cloudFunctionClient';
+
+describe('CloudFunction', function () {
```

Comment:
Could this be an after statement in the e2e test file?

Reply:
This is used with the e2e test where I invoke the action directly, not via test runner.